app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/pocket/PocketStoriesComposables.kt

```diff
@@ -73,20 +81,35 @@ fun PocketStory(
         Card(
             elevation = 6.dp,
             shape = RoundedCornerShape(4.dp),
-            modifier = Modifier
-                .size(160.dp, 87.dp)
-                .padding(bottom = 8.dp)
+            modifier = Modifier.size(160.dp, 87.dp)
         ) {
-            // Don't yet have a easy way to load URLs in Images.
-            // Default to a solid color to make it easy to appreciate dimensions
-            Box(Modifier.background(Color.Blue))
-            // Image(
-            //     painterResource(R.drawable.ic_pdd),
-            //     contentDescription = "hero image",
-            //     contentScale = ContentScale.FillHeight,
-            // )
+            ImageLoader(
+                client = client,
+                // The endpoint allows us to ask for the optimal resolution image.
+                url = story.imageUrl.replace(
+                    "{wh}",
+                    with(LocalDensity.current) {
+                        "${160.dp.toPx().roundToInt()}x${87.dp.toPx().roundToInt()}"
+                    }
+                )
+            ) {
+
+                Placeholder {
+                    Shimmer()
+                }
+
+                WithImage { painter ->
+                    Image(
+                        painter,
+                        modifier = Modifier.size(160.dp, 87.dp),
+                        contentDescription = "${story.title} story image"
+                    )
+                }
+            }
```

Comment:
nit: This whole block could almost be its own `PocketImage` kind of composable?

Reply:
👍. I like it. 
Was talking with Nicole and Gabriel about sharing the composables with the "Jump back in" items and generally about making the Composables reusable so while these were more to test the basics work for the final design I'll work on breaking off individual Composables that can be reused and recombined for other features.